src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.controller;
+
+import javax.validation.Valid;
+import mate.academy.spring.model.OperationResult;
+import mate.academy.spring.model.Result;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
```

Comment:
You have to use DtoResponseMapper in this constructor as well (specified by UserResponseDto and User).